docs-src/concepts/what-is-an-action.md

```diff
@@ -0,0 +1,46 @@
+---
+id: what-is-an-action
+title: What is an Action?
+sidebar_label: Action
+description: An Action is the fundamental pricing unit in Temporal Cloud. Each Action is an Event in the Event History of a Workflow Execution.
+tags:
+  - term
+  - explanation
+---
+
+An Action is the fundamental pricing unit in [Temporal Cloud](/concepts/what-is-temporal-cloud).
+Each Action is an [Event](/concepts/what-is-an-event) in the [Event History](/concepts/what-is-an-event-history) of a [Workflow Execution](/concepts/what-is-a-workflow-execution).
+(Be aware that not all Events are Actions.)
+
+**Workflow Actions**
+
+- Start Workflow Execution.
+- Reset Workflow Execution.
```

Comment:
Probably obvious, but can I get confirmation that the actions that happened before reset are still counted? This is another one that's hard to quantify afterwards because the actions are gone.